.changesets/README.md

```diff
@@ -10,10 +10,19 @@ This directory keeps files which individually represent entries that will repres
 >
 > The aforementioned **tooling doesn't exist yet** but will be created soon. 😺
 
-Create a file **by hand** in this directory for each individual changelog entry by using the required convention.  That convention is:
+### How to create a Changelog entry
```

Comment:
Is it worth preserving the instructions for creating a changelog by hand or at least noting that it's still possible?

Reply:
I don't think so but those instructions are preserved in the form of convention which is still below.  I think we should have a strong bias for using the tooling and correcting it if it's not working.